package.json

```diff
@@ -90,7 +90,7 @@
     "@chatie/git-scripts": "^0.6.2",
     "@chatie/semver": "^0.4.7",
     "@chatie/tsconfig": "^4.6.2",
-    "@swc/core": "^1.2.113",
+    "@swc/core": "1.3.44",
```

Comment:
I think `@swc/*` can be included in the `@chatei/tsconfig` so that all repo can easily be managed?

How do you think?